com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -10,21 +10,31 @@ namespace Unity.Netcode
     // Might include tick in a future milestone, to address past variable value
```

Comment:
I can understand that spawn/despawn have to be part of the SnapshotSystem. (even though I'd prefer them to be separate 😃). But could we consider maybe moving them into a partial class for better code separation and visibility. (What I mean is make SnapshotSystem partial then create SnapshotSystem.Spawning.cs and add a partial SnapshotSystem class to it and move all spawning functions over.

Reply:
MTT-1099
:-) that's the best I can offer at the moment. Acknowledging it, and remembering to come back to it.